README.md

```diff
@@ -2,14 +2,15 @@
 
 # PeopleDepot
 
-PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot provides backend infrastructure to provide a single source of truth on people, program areas, and project data for all Hack for LA projects. PeopleDepot uses a Django database and Django REST Framework that acts as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA, Civic Tech Index, Tables).
+PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot aims to provide a single source of truth as the backend infrastructure and database store for all Hack for LA projects, including data about people, program areas, and projects. PeopleDepot uses Django for its database and REST Framework and serves as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA Website, Civic Tech Index, Tables).
```

Comment:
The changes look good.

Not directly related to the current changes, 

1. We're only the data store for the organizational (or some word that means projects dealing with data that has to do with the internal infrastructure) projects for Hack for LA, not "all Hack for LA projects".

2. We're using PostgreSQL for the database, Django as the backend data model framework with Django REST Framework for the API layer.

Reply:
I think it'd be good to improve the wording! Let me give it some thought.

Reply:
Taking your recommendation, what do you think about this?

> PeopleDepot is a project of Hack for LA, a brigade of Code for America. PeopleDepot aims to provide a single source of truth as the backend infrastructure and **_data store for Hack for LA projects_**, including data about people, program areas, and projects. **_PeopleDepot uses PostgreSQL for its database management and Django as the backend data model framework with Django REST Framework for the API layer. PeopleDepot's goal is to serve_** as a repository of information for other infrastructure projects (e.g., VRMS, Hack for LA Website, Civic Tech Index, Tables, etc).

Reply:
looks good